http.go

```diff
@@ -20,17 +21,29 @@ type URLReq struct {
 
 func (ur *URLReq) RequestUrl() (*http.Response, error) {
 
+	duration := 5000 * time.Millisecond
+
+	// Check if a timeout header is specified
+	if timeout, ok := ur.Headers["TimeoutDuration"]; ok && timeout != "" {
```

Comment:
ganti headers jadi ini Request-Timeout

[https://www.ietf.org/archive/id/draft-thomson-hybi-http-timeout-00.html#:~:text=Request%2DTimeout%20Header,-The%20Request%2DTimeout&text=A%20client%20adds%20a%20Request,single%20integer%20value%20in%20seconds.](url)